tests/onnx/quantization/helper.py

```diff
@@ -0,0 +1,117 @@
+TENSOR_TYPE_MAP = {
+    int(TensorProto.FLOAT): TensorDtypeMap(
+        np.dtype("float32"), int(TensorProto.FLOAT), "TensorProto.FLOAT"
+    ),
+    int(TensorProto.UINT8): TensorDtypeMap(
+        np.dtype("uint8"), int(TensorProto.INT32), "TensorProto.UINT8"
+    ),
+    int(TensorProto.INT8): TensorDtypeMap(
+        np.dtype("int8"), int(TensorProto.INT32), "TensorProto.INT8"
+    ),
+    int(TensorProto.UINT16): TensorDtypeMap(
+        np.dtype("uint16"), int(TensorProto.INT32), "TensorProto.UINT16"
+    ),
+    int(TensorProto.INT16): TensorDtypeMap(
+        np.dtype("int16"), int(TensorProto.INT32), "TensorProto.INT16"
+    ),
+    int(TensorProto.INT32): TensorDtypeMap(
+        np.dtype("int32"), int(TensorProto.INT32), "TensorProto.INT32"
+    ),
+    int(TensorProto.INT64): TensorDtypeMap(
+        np.dtype("int64"), int(TensorProto.INT64), "TensorProto.INT64"
+    ),
+    int(TensorProto.BOOL): TensorDtypeMap(
+        np.dtype("bool"), int(TensorProto.INT32), "TensorProto.BOOL"
+    ),
+    int(TensorProto.FLOAT16): TensorDtypeMap(
+        np.dtype("float16"), int(TensorProto.UINT16), "TensorProto.FLOAT16"
+    ),
+    # Native numpy does not support bfloat16 so now use float32 for bf16 values
+    # TODO ONNX should dirtectly use bfloat16 for bf16 values after numpy has supported bfloat16 type
+    int(TensorProto.BFLOAT16): TensorDtypeMap(
+        np.dtype("float32"), int(TensorProto.UINT16), "TensorProto.BFLOAT16"
+    ),
+    int(TensorProto.DOUBLE): TensorDtypeMap(
+        np.dtype("float64"), int(TensorProto.DOUBLE), "TensorProto.DOUBLE"
+    ),
+    int(TensorProto.COMPLEX64): TensorDtypeMap(
+        np.dtype("complex64"), int(TensorProto.FLOAT), "TensorProto.COMPLEX64"
+    ),
+    int(TensorProto.COMPLEX128): TensorDtypeMap(
+        np.dtype("complex128"), int(TensorProto.DOUBLE), "TensorProto.COMPLEX128"
+    ),
+    int(TensorProto.UINT32): TensorDtypeMap(
+        np.dtype("uint32"), int(TensorProto.UINT32), "TensorProto.UINT32"
+    ),
+    int(TensorProto.UINT64): TensorDtypeMap(
+        np.dtype("uint64"), int(TensorProto.UINT64), "TensorProto.UINT64"
+    ),
+    int(TensorProto.STRING): TensorDtypeMap(
+        np.dtype("object"), int(TensorProto.STRING), "TensorProto.STRING"
+    ),
+}
+
+
+def load_model_topology_with_random_weights(model_path: Union[str, Path]) -> onnx.ModelProto:
+    """
+    Loads onnx model and fills the all external tensors by random values.
+
+    :param model_path: Path to the onnx model to load.
+    :return: Onnx model with filled the all external tensors by random values.
+    """
+    model = onnx.load_model(model_path, load_external_data=False)
+    onnx_graph = ONNXGraph(model)
+    for tensor in onnx_graph.onnx_model.graph.initializer:
+        if uses_external_data(tensor):
+            np_dtype = TENSOR_TYPE_MAP[tensor.data_type].np_dtype
+            np_tensor = np.random.random(list(tensor.dims)).astype(np_dtype)
```

Comment:
Set seed before using random

Reply:
Replace to loading zeros weights, not random. It is better for degerming whether the model was loaded by provided method